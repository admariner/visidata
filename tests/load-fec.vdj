#!vd -p
{"sheet": null, "col": null, "row": null, "longname": "open-file", "input": "sample_data/1794685.fec", "keystrokes": "o", "comment": null}
{"sheet": "1794685", "col": "", "row": "\u30adheader", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open current row with sheet-specific dive"}
