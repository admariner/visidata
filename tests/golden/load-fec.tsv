key	value
record_type	HDR
ef_type	FEC
fec_version	8.4
soft_name	Campaign Manager 360
soft_ver	1.0
report_id	FEC-1724438
report_number	6
comment	
